installer/install.iss

```diff
@@ -899,6 +911,27 @@ begin
 #endif
 end;
 
+function IsUpgrade(CurrentVersion,PreviousVersion:String):Boolean;
+begin
+    Result:=(not (CurrentVersion=PreviousVersion) and not IsDowngrade(CurrentVersion,PreviousVersion))
+end;
+
+{ Represent a set as a string of comma-separated values }
+function IsInSet(ASet:String;Value:Integer):Boolean;
+begin
+    Result:=(Pos(','+IntToStr(Value)+',',ASet)>0);
+end;
+
+function AddToSet(ASet:String;Value:Integer):String;
```

Comment:
How about making this a `procedure` and turning `ASet` into a `var` (i.e. passing it by reference rather than by value)?